tasks/86_working-hours/working-hours.js

```diff
@@ -0,0 +1,38 @@
+export function formatWorkingHours(arr) {
+  let start = null;
+  let end = null;
+  const tillDay = () => start !== end ? ` - ${end.day.toUpperCase()}` : '';
+  const dayAndHours = () => {
+    let startDay = start.day.toUpperCase();
+    return `${startDay}${tillDay()}: ${start.from} - ${start.to}`;
+  };
+  const daysOfWeek = ['mon', 'tue', 'wed', 'thu', 'fri', 'sat', 'sun'];
+  const sortedArray = arr.sort((a, b) => {
+    const indexOfEl = (day) => daysOfWeek.indexOf(day);
+    if (indexOfEl(a.day) < indexOfEl(b.day)) {
+      return -1;
+    } else {
+      return 1;
+    }
+  });
+
+  let scheduleArray = [];
+  for (let i = 0; i < sortedArray.length; i++) {
+    if (start === null) {
```

Comment:
as it needs only for first iteration maybe better already prepare you variable for this and start your loop from 1 and not 0?